### ISSUE TITLE
Unable to claim plex server

### ISSUE THREAD

**Issue by ltheakston** [kelinger/OmniStream#25]
I previously used gooby, but decided to switch to omnistream. I tried just upgrading first but ran into lots of issues, so instead I did a fresh install. Everything is up and running, but when I tried to claim the plex server, it didn't work. 
I entered the claim token after running setup-plex, immediately went to plex.mydomain, but I just got the web player screen with no server. 
I am getting infuriated by this and am giving in to the wisdom of others.

**Comment by TechPerplexed** [comment]
Right, documentation is still a little sketchy, sorry about that. So just to verify, did you install Plex before running `setup-plex`?

**Comment by ltheakston** [comment]
Yes, plex was fully up and running before I ran setup-plex.

**Comment by wickedshrapnel** [comment]
After you generate a token you have to run option 2 "Start minimal OmniStream" before you can claim your plex to take it out of the cloudflare proxy. Just run all those setup plex options in order. 1 get token 2 minimal.. now go claim your plex at plex. Domain. 3 come back and clear token 4 restart in production mode which put back in proxy mode.

**Comment by ltheakston** [comment]
Just tried that, after starting minimal omnistream, plex.mydomain gives me a 404 error.

**Comment by TechPerplexed** [comment]
Give it some time... it'll take a few minutes to come up.

**Comment by ltheakston** [comment]
I'm still getting the 404 error and it's past the 4 minutes

**Comment by wickedshrapnel** [comment]
try running omni up after running step 2 to make sure plex is up and DNS records made uncached.

**Comment by wickedshrapnel** [comment]
might have to run omni down after step 2 to clear out the cached record and omni up to create the new one.

**Comment by ltheakston** [comment]
That worked, thanks. I had to re-enable all the other containers though, as restart production mode didn't work.

**Comment by wickedshrapnel** [comment]
Cool
